Modules/_ctypes/callproc.c

```diff
@@ -498,9 +498,17 @@ PyCArg_repr(PyCArgObject *self)
         break;
 
     case 'c':
-        sprintf(buffer, "<cparam '%c' (%c)>",
-            self->tag, self->value.c);
-        break;
+        if ((unsigned char)self->value.c < 128
+            && _PyUnicode_IsPrintable(self->value.c)
```

Comment:
Is printability the right test here? Tab, newline, carriage return, vertical tab, form feed are all printable but probably don't make sense in this context.

Reply:
`'\t'.isprintable()` returns False.